cogs/word_filter.py

```diff
@@ -340,6 +340,13 @@ def _updateSettings(self):
             self.whitelist.update(myDict)
             self._updateWhitelist()
 
+        pattern = r'<#(\d+)>'
+        match = re.search(pattern, channelName)

```

Comment:
Perhaps we can look into using [discord.utils.get](https://discordpy.readthedocs.io/en/latest/api.html#discord.utils.get) to see if it's a channel object?